activerecord/lib/active_record/relation/batches.rb

```diff
@@ -98,12 +106,19 @@ def find_each(start: nil, end_at: nil, batch_size: 1000)
     #
     # NOTE: You can't set the limit either, that's used to control
     # the batch sizes.
-    def find_in_batches(start: nil, end_at: nil, batch_size: 1000)
-      relation = self
+    def find_in_batches(begin_at: nil, end_at: nil, batch_size: 1000, start: nil)
+      if start
+        begin_at = start
+        ActiveSupport::Deprecation.warn(<<-MSG.squish)
+            Passing `start` value to find_in_batches is deprecated.
```

Comment:
Also say that it will be removed in Rails 5.1.

@chancancode is this the correct format for multi line comments? I can never remember :sweat_smile: